packages/radio-group/src/vaadin-radio-group.js

```diff
@@ -73,510 +85,380 @@ class RadioGroupElement extends ThemableMixin(DirMixin(PolymerElement)) {
           display: none !important;
         }
 
-        .vaadin-group-field-container {
+        [part='container'] {
           display: flex;
           flex-direction: column;
-
-          /* Prevent horizontal overflow in IE 11 instead of wrapping radios */
-          width: 100%;
         }
 
-        [part='label']:empty {
+        :host(:not([has-label])) [part='label'] {
           display: none;
         }
       </style>
-
-      <div class="vaadin-group-field-container">
-        <label part="label">[[label]]</label>
+      <div part="container">
+        <div part="label">
+          <slot name="label"></slot>
+          <span part="required-indicator" aria-hidden="true"></span>
+        </div>
 
         <div part="group-field">
-          <slot id="slot"></slot>
+          <slot></slot>
         </div>
 
-        <div
-          part="helper-text"
-          id="[[_helperTextId]]"
-          aria-live="assertive"
-          aria-hidden$="[[_getHelperTextAriaHidden(helperText, _helperTextId, _hasSlottedHelper)]]"
-        >
-          <slot name="helper">[[helperText]]</slot>
+        <div part="helper-text">
+          <slot name="helper"></slot>
         </div>
 
-        <div
-          part="error-message"
-          id="[[_errorId]]"
-          aria-live="assertive"
-          aria-hidden$="[[_getErrorMessageAriaHidden(invalid, errorMessage, _errorId)]]"
-          >[[errorMessage]]</div
-        >
+        <div part="error-message">
+          <slot name="error-message"></slot>
+        </div>
       </div>
     `;
   }
 
-  static get is() {
-    return 'vaadin-radio-group';
-  }
-
   static get properties() {
     return {
       /**
-       * The current disabled state of the radio group. True if group and all internal radio buttons are disabled.
+       * The value of the radio group.
+       *
+       * @attr {string} value
```

Comment:
Please remove `@attr` annotation, we don't need it for non-camel-case properties.

Reply:
Done.